docs/firefighting/index.md

```diff
@@ -0,0 +1,108 @@
+This page summarizes how to do various things related to HQ.
+For a more user-friendly guide check out [Cory's brown bag on the topic](http://prezi.com/wedwm-dgqto7/firefighting-hq/).
+
+# HQ Architecture and Machines
+
+![](./hq_architecture.jpg)
```

Comment:
Not sure if this will actually work. I don't have ruby on my local machine, os haven't been able to test it out

Reply:
This should work—I've done something similar [here](https://github.com/millerdev/lms-ultralight/blob/master/README.md). Maybe try this:

```suggestion
![HQ Architecture and Machines](hq_architecture.jpg?raw=true "HQ Architecture and Machines")
```

Reply:
It turns out that what I did does work if I use the right image extension https://github.com/dimagi/commcare-cloud/blob/5f8e72de44046e927d5c0d8382ed3bb44045c8ef/docs/firefighting/index.md